pkg/uninstall/uninstall.go

```diff
@@ -115,6 +119,24 @@ This will uninstall Everest and all monitoring resources deployed by it. All oth
 	return nil
 }
 
+func (u *Uninstall) areResourcesExist(ctx context.Context) error {
```

Comment:
The name of the function is weird, how about
```suggestion
func (u *Uninstall) checkResourcesExist(ctx context.Context) error {
```

Reply:
Make sense. Changed it. Thanks